Drivers/Src/MotorController.c

```diff
@@ -70,6 +70,36 @@ ErrorStatus MotorController_Read(CANbuff *message){
         }
         message->firstNum = firstSum;
         message->secondNum = secondSum;
+
+        //read first number (bits 0-31) to get motor error issues
+        uint32_t maskMotorTemp=64; //check if motor temperature is an issue
```

Comment:
Probably should define this as a constant somewhere (perhaps at the top of the file). It would also be a bit nicer to define it as 1<<6 instead of 64, to make it clear we're checking the sixth bit.